config/v1/config.go

```diff
@@ -0,0 +1,10 @@
+package v1
+
+type Config struct {
+	Integration string
+	Vcs         struct {
+		Repos   []string
+		Options interface{}
+	}
+	Services []interface{}
```

Comment:
I made Services and VCS Options interfaces so that Github/Gitlab/Bamboo/Travis/whatever can define their own settings; in practice this turned out wonky

Examples
https://github.com/de1ux/prowler/pull/11/files#diff-783c8ea570fa7d19c492e7374dbce6c4R21
https://github.com/de1ux/prowler/pull/11/files#diff-d9f238a11ad87b60a803996780f611d6R16

@bign8 I'd love your thoughts on this